frontend/public/components/machine-set.tsx

```diff
@@ -202,6 +203,7 @@ export const MachineTabPage: React.SFC<MachineTabPageProps> = ({
 const MachineSetDetails: React.SFC<MachineSetDetailsProps> = ({ obj }) => {
   const machineRole = getMachineRole(obj);
   const { availabilityZone, region } = getMachineAWSPlacement(obj);
+  const instanceType = obj.spec?.template?.spec?.providerSpec?.value?.instanceType;
```

Comment:
This might be AWS specific

cc @JoelSpeed

Reply:
I added logic for the providers we have data for and noted where data is not available.